scripts/create-fake-data.js

```diff
@@ -0,0 +1,207 @@
+
+let signupIndex = 0;
```

Comment:
running better-npm-run in /Users/pekkalammi/www/ilmo2
Executing script: create-fake-data

to be executed: node scripts/create-fake-data
/Users/pekkalammi/www/ilmo2/scripts/create-fake-data.js:165
let signupIndex = 0;
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
    at node.js:405:3
error Command failed with exit code 1.